packages/teleport/src/AppLauncher/useAppLauncher.ts

```diff
@@ -47,6 +47,7 @@ function resolveRedirectUrl(params: UrlLauncherParams) {
   const location = window.location;
   const port = location.port ? ':' + location.port : '';
   const state = getUrlParameter('state', location.search);
+  const path = getUrlParameter('path', location.search);
```

Comment:
Could you use something more explicit like `redirect` or `redirect-path`?

Reply:
Because this param only exists when an app is requested before authenticated, renaming the param in the query string itself might be a bit confusing since a `redirect_uri` already exists, but renaming in the context of this file + the comments are a great idea, thanks!